internal/cluster/validator.go

```diff
@@ -400,6 +401,26 @@ func (v *clusterValidator) allHostsAreReadyToInstall(c *clusterPreprocessContext
 	return ValidationFailure, "The cluster has hosts that are not ready to install."
 }
 
+func (v *clusterValidator) OciManifestsSatisfied(c *clusterPreprocessContext) (ValidationStatus, string) {
+	// If cluster platform type is not OCI ignore that validation
+	if !(c.cluster.Platform != nil && common.PlatformTypeValue(c.cluster.Platform.Type) == models.PlatformTypeOci) {
+		return ValidationSuccess, "OCI platform wasn't set"
+	}
+
+	usages, err := usage.Unmarshal(c.cluster.Cluster.FeatureUsage)
+	if err != nil {
+		return ValidationFailure, "Failed to parse feature usages"
+	}
+
+	for _, usg := range usages {
+		if usg.Name == usage.CustomManifest {
+			return ValidationSuccess, "Found custom manifests"
+		}
+	}
+
+	return ValidationFailure, "Can't find custom manifests"
```

Comment:
We need to add here what needs to be done.
kaedward@redhat.com FYI

Reply:
How about "The custom manifests required for OCI platform integration have not been added"

Reply:
Is "OCI platform integration" something that the user will understand? If it is then go ahead and keep it, otherwise I think it can be taken out to avoid confusion.
**Cluster installation failed**
The custom manifest required for installation has not been added. Add a custom manifest to BLANK and try again"

Reply:
Some context and clarifications:

- This message should be shown only in the CLI (because in the UI you can't proceed in the wizard if you don't provide manifests
- This message is not about a failure of a cluster installation, the cluster installation was not triggered yet
- I don't think we should call it "OCI" as we call the integration "Oracle" in the UI drop-down. If we want to stick with "OCI", we should also adjust the UI.

Reply:
"Oracle" is too generic - it should be "Oracle Cloud Infrastructure"

Reply:
Regarding the message here, how about:
The custom manifests required for OCI platform integration have not been added. Add the custom manifests and try again.

Reply:
Then it should be the same in the UI.

Reply:
So what is the decision?

Reply:
"The custom manifest required for Oracle Cloud Infrastructure platform integration has not been added. Add a custom manifest to continue."